alembic/env.py

```diff
@@ -0,0 +1,77 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+import gdc_ng_models.models.notifications as notifications_models
+
+target_metadata = notifications_models.Base.metadata
```

Comment:
I think we first need to refactor all the models (misc, qcreport etc) to use the same `Base` and use that here instead. That will introduce a some more work across other projects that make use of the models